src/packages/users/current-user/modals/current-user/current-user-modal.element.ts

```diff
@@ -25,15 +32,16 @@ export class UmbCurrentUserModalElement extends UmbLitElement {
 			this._observeCurrentUser();
 		});
 
-		this.consumeContext(UMB_AUTH, (instance) => {
-			this.#auth = instance;
-		});
-
 		this.consumeContext(UMB_APP, (instance) => {
 			this.#appContext = instance;
 		});
 
-		this._observeCurrentUser();
+		this.consumeContext(UMB_LOCALIZATION_CONTEXT, (instance) => {
+			instance.localizeMany(['general_close', 'general_logout']).subscribe((values) => {
```

Comment:
Wouldn't it be nicer to use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for the `values` array, so you don't have to use indexes (that are very prone to typos and make it harder to add new keys before/between existing ones?

Reply:
If it's possible to provide direct mappings, that would make it even better... Something like:
```ts
instance.localizeMap({
  'general_close': value => this.labelClose = value,
  'general_logout': value => this.labelLogout = value
});
```

Reply:
Yes, but that is not backwards compatible with the current backoffice. The idea was that people had to rework as little as possible. Although, now we are looking into a different API so things might change and we probably won't need methods such as `localizeMany` moving forward.